app/models/manageiq/providers/cloud_manager/provision_workflow.rb

```diff
@@ -90,6 +88,19 @@ def allowed_ci(ci, relats, filtered_ids = nil)
     super(ci, relats, sources, filtered_ids)
   end
 
+  def availability_zone_to_cloud_network(src)
+    if src[:availability_zone]
+      load_ar_obj(src[:availability_zone]).cloud_subnets.each_with_object({}) do |cs, hash|
+        cn = cs.cloud_network
+        hash[cn.id] = cn.name
+      end
+    else
+      load_ar_obj(src[:ems]).cloud_subnets.collect(&:cloud_network).each_with_object({}) do |cn, hash|
+        hash[cn.id] = cn.name
+      end
+    end
+  end
+
   def get_source_and_targets(refresh = false)
```

Comment:
@d-m-u - from reading this method it sounds like this loads the cloud_networks associated with the availability zone of the source which is contrary to the PR description, yay or nay?

Reply:
Yeah. Fixed. Sorry.